runtime-cryptography/openssl-1.1/autobuild/patches/0001-patch-Configure-for-loongson3.patch

```diff
@@ -0,0 +1,25 @@
+From 7bc94b7861c09c5307af4dfcfdb6a3439b4d37bc Mon Sep 17 00:00:00 2001
+From: Student Main <studentmain@aosc.io>
+Date: Wed, 12 Feb 2025 13:27:40 +0800
+Subject: [PATCH] patch Configure for loongson3
+
+---
+ Configure | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/Configure b/Configure
+index 78cc15d..d6484d5 100755
+--- a/Configure
++++ b/Configure
+@@ -1248,7 +1248,7 @@ if ($target =~ /linux.*-mips/ && !$disabled{asm}
+         # minimally required architecture flags for assembly modules
+         my $value;
+         $value = '-mips2' if ($target =~ /mips32/);
+-        $value = '-mips3' if ($target =~ /mips64/);
++        $value = '-mips64r2' if ($target =~ /mips64/);
```

Comment:
Please make this architecturally conditional - our other mips64el port, loongson2f, is mips3.

Reply:
Also, would this be something to do also for openssl (the main 3.x package)? cc @chenx97 @FlyGoat @rickliu2000